framework/Sources/VGSFramework/Core/APIClient.swift

```diff
@@ -21,7 +21,7 @@ class APIClient {
         baseURL = url
     }
     
-    func sendRequest(path: String, value: BodyData, completion block: @escaping (_ data: JsonData?, _ error: Error?) -> Void) {
+    func sendRequest(path: String, method: HTTPMethod = .post, value: BodyData, completion block: @escaping (_ data: JsonData?, _ error: Error?) -> Void) {
```

Comment:
Is it a typical record for swift? 
`method: HTTPMethod = .post`

can we change it on 
`method: HTTPMethod = .POST`